src/sentry/ingest/billing_metrics_consumer.py

```diff
@@ -199,6 +200,10 @@ def _bulk_commit(self) -> None:
         """Commits and clears the ready to commit queue."""
         if not self._ready_to_commit:
             return
+        metrics.incr(
```

Comment:
I think we have to use [`metrics.gauge`](https://github.com/getsentry/sentry/blob/0654a3c6b741f3312e3ee5c61801bb35b9c1b713/src/sentry/metrics/base.py#L29) instead of `metrics.incr`, because you don't want to sum up queue sizes.